server/channels/app/plugin_test.go

```diff
@@ -805,202 +809,559 @@ func TestPluginStatusActivateError(t *testing.T) {
 	})
 }
 
-func TestProcessPrepackagedPlugins(t *testing.T) {
-	th := Setup(t)
-	defer th.TearDown()
+type byId []*plugin.PrepackagedPlugin
+
+func (a byId) Len() int           { return len(a) }
+func (a byId) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
+func (a byId) Less(i, j int) bool { return a[i].Manifest.Id < a[j].Manifest.Id }
 
+type pluginStatusById model.PluginStatuses
+
+func (a pluginStatusById) Len() int           { return len(a) }
+func (a pluginStatusById) Swap(i, j int)      { a[i], a[j] = a[j], a[i] }
+func (a pluginStatusById) Less(i, j int) bool { return a[i].PluginId < a[j].PluginId }
+
+func TestProcessPrepackagedPlugins(t *testing.T) {
```

Comment:
I expanded on the base unit tests heavily -- with some refactoring for clarity -- before adding coverage for transitionally prepackaged plugins.